code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -528,6 +542,7 @@
 	var/mob/living/carbon/human/owner
 	var/list/datum/spellbook_entry/entries = list()
 	var/list/categories = list()
+	var/owner_is_apprentice = 0 // to see if the book belongs to an apprentice
```

Comment:
Yes but you can check if the special_role is apprentice or not, just like you can check if it's wizard or no. So if the wizard spawns it and tries to use it he'll get the below message. Similar to if the apprentice tries to read the wizard's book you can see "oh the owner's special_role is wizard..."